src/Overcrowdin/Overcrowdin.csproj

```diff
@@ -14,10 +14,11 @@
     <Description>.NET Core Global Tool providing a limited but useful Crowdin CLI. Built over the
 Crowdin.Api [Copyright (c) 2019 crowdin]</Description>
     <PackageLicenseUrl>https://opensource.org/licenses/MIT</PackageLicenseUrl>
-    <PackageProjectUrl></PackageProjectUrl>
+    <PackageProjectUrl>https://github.com/sillsdev/overcrowdin</PackageProjectUrl>
     <RepositoryUrl>https://github.com/sillsdev/overcrowdin</RepositoryUrl>
     <PackageTags>.NET Core;l10n;localization;cli</PackageTags>
     <PackageReleaseNotes>First version includes support for generating a configuration file from a crowdin project, adding source files to crowdin, updating source files in crowdin and downloading translation zip files from crowdin.</PackageReleaseNotes>
+    <Version>1.0.0</Version>
```

Comment:
Would be better to use GItVersion - that way the version number automatically increments.

Reply:
Yes, I should figure out how to use GitVersion and do that.

Reply:
That's easy to get started: 

- copy `GitVersion.yml` from SIL.NuGetCleaner
- add `<PackageReference Include="GitVersionTask" Version="5.0.1" />` to the .csproj file
- remove `<Version>1.0.0</Version>` from .csproj
- add a `v1.0.0` git tag (or add `next-version: 1.1.0` to `GitVersion.yml`)

This will create pre-release packages on every commit on master. To release a new version create and push a git tag with the version number.